src/main/cpp/utility/BetterSubsystemBase.cpp

```diff
@@ -0,0 +1,44 @@
+#include "utility/BetterSubsystemBase.h"
+#include <iostream>
+
+void BetterSubsystemBase::AddPID(MotorUtils::Motor motor) {
+  std::vector<MotorUtils::Motor> motorVec {motor};
+  AddPID(motorVec);
+}
+
+void BetterSubsystemBase::AddPID(std::vector<MotorUtils::Motor> motors) {
+  this->motors = motors;
+
+  for (int i = 0; i < this->motors.size(); i++) {
```

Comment:
You can use `for (auto &i: this->motors)` instead for brevity and readability. You don't need to change this but it's just good to know.